### DISCUSSION TITLE
Total overall uno download size?

### DISCUSSION THREAD

**Discussion by renhiyama** [unoplatform/uno#8025] [Q&A]
Hello there, since I am on a metered network, with slow connection speed, I would like to learn about the download size of uno platform (on linux)
So if the download size is a lot much for me, I would like to look for alternatives that are small in size.

**Comment by jeromelaban** [comment]
The size of the package depends on what your application is doing. The more code you add, the larger your application will get.

Uno supports [IL trimming](https://docs.microsoft.com/en-us/dotnet/core/deploying/trimming/trimming-options), but the largest contributor in size is always the runtime. If you enable self-contained mode, the package will get larger, but if you install the runtime on the target machine, your package will be smaller.

**Comment by renhiyama** [comment]
Hey @jeromelaban thanks for answering, but I wanted to know the download size of uno platform SDK for me, for developing apps. I didn't talked about publishing apps for others and discussing about the download size of my app made with uno for others. The problem is that, dotnet seems to be non informative most of the times, like it doesn't even shows the download size of uno

**Reply by jeromelaban** [reply]
The size of the SDKs download varies across target platforms, development platforms, versions and IDEs being used to create apps. 

There are many dependencies that are not our own, for which the size can vary greatly. I suggest you try it for you own environment configuration and see what download size you get.
